### DISCUSSION TITLE
Share labels flags screenshote to another component

### DISCUSSION THREAD

**Discussion by ilkarataev** [WeblateOrg/weblate#4867] [General]
Hello, our project workflow need to work with all source information like labels,scrennshots_context,explanations e.t.c. to all component that can have different branch. How can we used it?

**Comment by nijel** [comment]
You can automate this using API, there is no built-in support for this.

**Reply by lacroixdavid1** [reply]
@nijel I can't find the API documentation to update source information labels. Is that implemented?

**Reply by nijel** [reply]
Indeed labels are currently not covered by the API, see https://github.com/WeblateOrg/weblate/issues/5275
